Patterns/German/German-DateTime.yaml

```diff
@@ -966,7 +966,6 @@ HolidayNames: !dictionary
     girlsday: [ girlsday ]
     whiteloverday: [ whiteloverday ]
     loverday: [ loverday ]
-    christmas: [ weihnachten, weihnachtstag ]
```

Comment:
Why this?

Reply:
There’s already a `firstchristmasday` entry containing the same values.